index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/b08094b2a2.js" crossorigin="anonymous"></script>
+    <title>Personal Portfolio</title>
+</head>
+
+<body>
+    <div>
```

Comment:
- [ ] Great job implementing the `the toolbar (or header):` section. :+1: :100:  But I think there is still room for some improvements.
- [ ] Following the best `HTML`  practices, you should wrap this with the `header` semantic tag. :+1: This is great as it follows the best industry standard. :smile: 
> Use the appropriate tags for each element (e.g., links, titles, etc.) and use HTML5 semantic tags (e.g., header, nav) over divs.

Reply:
> * [ ]  Great job implementing the `the toolbar (or header):` section. 👍 💯  But I think there is still room for some improvements.
> * [ ]  Following the best `HTML`  practices, you should wrap this with the `header` semantic tag. 👍 This is great as it follows the best industry standard. 😄
> 
> > Use the appropriate tags for each element (e.g., links, titles, etc.) and use HTML5 semantic tags (e.g., header, nav) over divs.

- [ ] Kindly note that the header of the page is where the navigation is, as the previous reviewer recommended, kindly replace the div tag with header tag for semantic reasons.